libs/langchain/langchain/vectorstores/llm_rails.py

```diff
@@ -47,14 +41,11 @@ def __init__(
 
         self._session = requests.Session()  # to reuse connections
         self.datastore_id = datastore_id
-        self.base_url = "https://api.llmrails.com/v1"
```

Comment:
should this be user configurable?

Reply:
No it should not missed it when developing fixing now